src/OpenTracing/GlobalTracer.php

```diff
@@ -18,7 +18,7 @@
      *
      * @param Tracer $tracer
      */
-    public static function setGlobalTracer(Tracer $tracer)
+    public static function withGlobalTracer(Tracer $tracer)
```

Comment:
Why `with`? `with` is mostly used for value objects where the `with*` methods return a new object (see PSR7), but this is a mutation/state change/side effect